include/rdma/fi_domain.h

```diff
@@ -341,7 +341,9 @@ static inline int
 fi_mr_refresh(struct fid_mr *mr, const struct iovec *iov, size_t count,
 	      uint64_t flags)
 {
-	struct fi_mr_modify modify = {0};
+	struct fi_mr_modify modify = {0, 
+		{NULL, 0, 0, 0, 0, NULL, 0, NULL}
```

Comment:
Can this be {0, {0}}?

Reply:
The C compiler is happy with that, but the C++ compiler issues warnings for each of the members of the nested struct that are not initialized (instead of the outer one, so you get 10 warnings in place of 1)